src/odemis/gui/comp/viewport.py

```diff
@@ -1923,7 +1925,3 @@ def setView(self, view, tab_data):
         super().setView(view, tab_data)
         self.canvas.add_background_overlay(self._tab_data_model.main.background)
 
-    def _on_stream_play(self, is_playing):
```

Comment:
I think we still need that code. In the superclass, dragging is disabled, so if we pause the stream, we cannot move around the overview images anymore. Maybe instead of this, we could remove CAN_MOVE_STAGE and add CAN_DRAG to the abilities in the init of the class? This is not done automatically since we have a stage in this viewport for the crosshairoverlay.

Reply:
This is precisely what the `FeatureOverviewViewport.__init__()` does. It removes CAN_MOVE_STAGE (and CAN_DRAG is set by default). 
I did try the code with the FASTEM, in the overview tab. It's (still) possible to drag the view all the time.